RFCs/FS-1092-anonymous-type-tagged-unions.md

```diff
@@ -88,6 +88,8 @@ An anonymous type-tagged union type should only be considered for when a union i
 
 5. There is some identified, concrete, simply explained benefit over using labelled discriminated unions, e.g. "we have a simpler API with fewer overloads".
 
+6. There is no possibility that the type will end up being consumed by .Net languages other than F#.
```

Comment:
This is a bit strong - it might be reasonable to expose - but I know what you mean :)

Reply:
Similar thoughts, but it's a good thing to clarify for now. Maybe as the feature takes shape a bit more we'll have a more nuanced position